mmv1/products/datafusion/api.yaml

```diff
@@ -262,3 +262,26 @@ objects:
               The resource name of the Pub/Sub topic. Format: projects/{projectId}/topics/{topic_id}
             required: true
             input: true            
+      - !ruby/object:Api::Type::Array
+        name: 'accelerators'
+        description: |
+          List of accelerators enabled for this CDF instance.
+        item_type: !ruby/object:Api::Type::NestedObject
+          properties:
+            - !ruby/object:Api::Type::Enum
+              name: 'acceleratorType'
+              description: |
+                The type of an accelator for a CDF instance.
+              values:
+                - :CDC
+                - :HEALTHCARE
+                - :CCAI_INSIGHTS
+              required: true
+            - !ruby/object:Api::Type::Enum
+              name: 'state'
+              description: |
+                The type of an accelator for a CDF instance.
+              values:
+                - :ENABLED
+                - :DISABLED
+                - :UNKNOWN
```

Comment:
It doesn't look like `UNKNOWN` is a value that can be explicitly specified to by users. ([API Doc](https://cloud.google.com/data-fusion/docs/reference/rest/v1/projects.locations.instances#State_1): `UNKNOWN` - Indicates that accelerator state is currently unknown. Requests for enable, disable could be retried while in this state)
I've also tested it locally and setting `state = "UNKNOWN"` ended up causing an error (as expected). We'd likely want to remove this.